test/spec/CheckSpec.hs

```diff
@@ -109,6 +109,27 @@ spec = do
           stderr output `shouldContain` "hello world"
           exitCode output `shouldBe` ExitFailure 1
 
+        it "allows to use backtick syntax" $ \onTestFailureLog -> do
+          writeHaskellProject repoDir
+          writeFile
+            "garn.ts"
+            [i|
+              import * as garn from "#{repoDir}/ts/mod.ts"
+              import { nixRaw } from "#{repoDir}/ts/nix.ts";
+
+              export const haskell = garn.mkProject(
+                {
+                  description: "",
+                  defaultEnvironment: garn.emptyEnvironment,
+                },
+                {},
+              ).addCheck("test-check")`echo error; false`;
```

Comment:
Maybe this test should actually splat in a package to help document why we want to support this syntax still?

Reply:
Yeah, good call! I did that in a0f09c5f. And also had to actually make it work. :)